mozilla_django_oidc_db/models.py

```diff
@@ -205,6 +206,16 @@ class OpenIDConnectConfigBase(SingletonModel):
         ),
     )
 
+    userinfo_claims_source = models.CharField(
+        verbose_name=_("User information claims extracted from"),
```

Comment:
```suggestion
        verbose_name=_("user information claims extracted from"),
```
verbose_name is capitalized by django during form field generation from model field (help_text does not have that, so that requires a proper sentence grammar)